Traceback (most recent call last):
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Root_UI.py", line 32, in __call__
    return apply(self.func, args)
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 148, in runSeq
    photon_produciton_experiment=experiment)
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 1245, in __init__
    self.data_hander = Photon_production_buffered_data_handler(t_stirap_length = 0.5 * photon_produciton_experiment.waveform_length*10**6)
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 1460, in __init__
    self.poll_queue()
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 1475, in poll_queue
    target=self.__analyse_buffer())
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 1487, in __analyse_buffer
    self.completed_iterations = self.analysis_buffer[-1][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Root_UI.py", line 32, in __call__
    return apply(self.func, args)
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 148, in runSeq
    photon_produciton_experiment=experiment)
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 1245, in __init__
    self.data_hander = Photon_production_buffered_data_handler(t_stirap_length = 0.5 * photon_produciton_experiment.waveform_length*10**6)
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 1460, in __init__
    self.poll_queue()
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 1475, in poll_queue
    target=self.__analyse_buffer())
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 1487, in __analyse_buffer
    self.completed_iterations = self.analysis_buffer[-1][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Root_UI.py", line 32, in __call__
    return apply(self.func, args)
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 148, in runSeq
    photon_produciton_experiment=experiment)
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 1245, in __init__
    self.data_hander = Photon_production_buffered_data_handler(t_stirap_length = 0.5 * photon_produciton_experiment.waveform_length*10**6)
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 1460, in __init__
    self.poll_queue()
  File "C:\Users\apc\Documents\Python Scripts\Cold Control Heavy\Experimental_UI.py", line 1481, in poll_queue
    self.after(delay_ms, lambda: self.poll_queue(delay_ms))
AttributeError: 'Photon_production_buffered_data_handler' object has no attribute 'after'
